pkg/apis/crd/v1alpha1/types.go

```diff
@@ -61,3 +76,28 @@ type NetworkPolicyRecommendationList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []NetworkPolicyRecommendation `json:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type RecommendedNetworkPolicy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec RecommendedNetworkPolicySpec `json:"spec,omitempty"`
+}
+
+type RecommendedNetworkPolicySpec struct {
+	Id          string      `json:"id,omitempty"`
+	Type        string      `json:"resultType,omitempty"`
+	TimeCreated metav1.Time `json:"timeCreated,omitempty"`
```

Comment:
I think RecommendedNetworkPolicySpec.TimeCreated and NetworkPolicyRecommendationStatus.EndTime are the same in Theia v0.2.0?
Do you think we need two similar fields here to track the end time? (No strong opinion, just curious when TimeCreated will be used.)

Reply:
In Theia v0.2.0 we use `TimeCreated` as the end time for those jobs if we do not find the corresponding SparkApplication but find the result in the ClickHouse. But in this release, I think we can always use the `NetworkPolicyRecommendation.Status.EndTime` for the end time.

I keep this field in RecommendedNetworkPolicy just for the consistency with the ClickHouse row. I also do not have a strong preference on whether to remove it.